architecture-building-systems/CEAforArcGIS
# I254 optimization to cea

This branch implements the resource optimization part of JPLUS (old research work) to the CEA. 

During the process, the demand algoerithm was updated. A new reference case study was also introduced.

steps for testing:
1....get case study of Zug (in the private repository) and copy to C:/

Then run:

2. .../demand/demand_main.py.
3. ... /optimization/optimization_main.py 

Then check after about 20 min that some files be created in the folder..

C:\reference-case-zug\baseline\outputs\data\optimization\master

If so..then it is working....
